datadog_lambda/tracing.py

```diff
@@ -283,6 +295,30 @@ def extract_context_from_sqs_or_sns_event_or_context(event, lambda_context):
         return extract_context_from_lambda_context(lambda_context)
 
 
+def _extract_context_from_eventbridge_sqs_event(event):
```

Comment:
Shouldn't this be the same as the `_extract_context_from_eventbridge_event`, only with a different body/input?

Reply:
I do agree. Nonetheless, if I used that method and failed, which is largely expected, it will directly try to extract the context from the lambda context. Instead of throwing an error so the next use case, `SNS => SQS`, is evaluated.

What do you think of this current approach? I would prefer rewriting the logic of the methods involved. Although I fear a refactor might take more pieces in account.

Reply:
This approach is good for now, thanks for the clarification!

Reply:
Definitely a note here to refactor the whole inferred spans.